lib/galaxy/schema/schema.py

```diff
@@ -3003,6 +3004,19 @@ class PageSummaryBase(BaseModel):
     )
 
 
+NotificationContentField: Optional[str] = Field(
+    default="",
+    title="Notification",
+    description="Raw text contents of the notification message.",
+)
+
+class NotificationBase(BaseModel):
+    message: str = Field(
```

Comment:
This should be message_text to be consistent with the database model.
You also need a response model with the properties you want to return. That can be the same model for now.